polyfill/lib/casts.mjs

```diff
@@ -37,7 +37,7 @@ export function CastAbsolute(arg, aux) {
   if ('bigint' === typeof arg) return new Absolute(arg);
   if ('string' === typeof arg) {
     try {
-      return Absolute.fromString(arg);
+      return Absolute.from(arg);
     } catch (ex) {}
```

Comment:
Catching the exception here only to then throw a new exception just before falling off the edge of the function makes no sense, and loses useful debugging information. Let's not do that.

Reply:
As mentioned, all the casts are going away in the next PR I make, so I decided not to go too deep on this. That said, I whole-heartedly agree.